### ISSUE TITLE
 NEI froze my game

### ISSUE THREAD

**Issue by Sphyix** [GTNewHorizons/GT-New-Horizons-Modpack#7481]
#### Which modpack version are you using?
2.1.0.0

#
#### If in multiplayer; On which server does this happen? 
delta

#
#### What did you try to do, and what did you expect to happen?
I swapped between moon, overworld and void dim to setup stuff on the planet.
After going back to my base, I noticed NEI wasn't searching anymore, so I tried to get my game to freeze in various ways but was unsuccessful, it froze when I used /back again to go to the moon.

Attaching thread dumps and logs.
[fml-client-1.log](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/files/5980405/fml-client-1.log)
[threaddump.zip](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/files/5980409/threaddump.zip)

And before anyone goes crazy, yes I updated the plugin to 2.0.9, but the file is renamed cause curseforge. I switched to multimc but forgot to rename it
Proof:
![notepad_XMa3uPamdk](https://user-images.githubusercontent.com/17811922/107924190-db3b8d00-6f72-11eb-88bc-19459a6be1fe.png)

**Comment by Sphyix** [comment]
Got a 2nd lock, this time at random, I wasn't switching dims or anything, just afk.
When I returned NEI stopped searching.
Idk if it can help, but I got a thread dump before locking up the game and after.
This time the only way of locking up was again switching dim.
[thread dump.zip](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/files/5981603/thread.dump.zip)

[fml-client-latest.log](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/files/5981620/fml-client-latest.log)
